src/protocol/coap.rs

```diff
@@ -62,6 +62,19 @@ pub struct Form {
     pub qblockwise: Option<BlockWiseTransferParameters>,
     #[serde(rename = "cov:hopLimit")]
     pub hop_limit: Option<u8>,
+    #[serde(rename = "cov:accept")]
+    pub accept: Option<u16>,
+    #[serde(rename = "cov:contentFormat")]
```

Comment:
We should mention https://www.iana.org/assignments/core-parameters/core-parameters.xhtml#content-formats somewhere, but that does not prevent the patch from landing.